scout/load/hpo.py

```diff
@@ -155,11 +155,12 @@ def _get_hpo_term_to_symbol(hpo_disease_lines: Iterable[str]) -> Dict:
     hpo_term_to_symbol = {}
     for hpo_to_symbol in parse_hpo_to_genes(hpo_disease_lines):
         hpo_id = hpo_to_symbol["hpo_id"]
+        hgnc_symbol = hpo_to_symbol["hgnc_symbol"]
 
         if hpo_id not in hpo_term_to_symbol:
-            hpo_term_to_symbol[hpo_id] = set([hpo_to_symbol["hgnc_symbol"]])
+            hpo_term_to_symbol[hpo_id] = set([hgnc_symbol])
         else:
-            hpo_term_to_symbol[hpo_id].update(hpo_to_symbol["hgnc_symbol"])
```

Comment:
I believe this was the like causing all those warnings. The warnings we didn't see before because the HPO terms had no associated genes!

Reply:
Aaah, thanks. The confusion there is likely since since there will eventually be many symbols to each id. But, there is still something weird with that `build_disease_term` loop for multiple entries now, since some genes get listed as missing even though they are in the db with the current symbol. E.g. "PWAR1" or "PRSS2" to name a couple that occur early on my list.

Reply:
No, all I have rechecked are either actually missing, bad gene names, like region approximations, or truly obsolete OR only on hg38. The latter must be a separate issue.